### PR TITLE
Last PR of stage to lesson rename!

### PR DESCRIPTION
**Pull request by dmcavoy**
This is it folks! The last things I could find to change from stage to lesson!

https://codedotorg.atlassian.net/browse/PLAT-981

### CODE REVIEW

File: apps/src/code-studio/hiddenLessonRedux.js [hunk 1]
```diff
@@ -195,7 +195,7 @@ export function getHiddenLessons(scriptName, canHideLessons) {
   return dispatch => {
     $.ajax({
       type: 'GET',
-      url: `/s/${scriptName}/hidden_stages`,
+      url: `/s/${scriptName}/hidden_lessons`,
```
Comment (bethanyaconnor) [inline apps/src/code-studio/hiddenLessonRedux.js:198]: When I go to /s/csd2-2021/hidden_lessons on production I get a 404 sad bee whereas /s/csd2-2021/hidden_stages works. I think the routes need to be updated?

Comment (dmcavoy) [reply apps/src/code-studio/hiddenLessonRedux.js:198]: On production or local?

Comment (bethanyaconnor) [reply apps/src/code-studio/hiddenLessonRedux.js:198]: 🤦‍♀️ I looked for the change in this PR and didn't see it but it's clearly there

**Review by bethanyaconnor — APPROVED** [review]

File: apps/src/code-studio/hiddenLessonRedux.js (hunk 1 above)
Comment (bethanyaconnor) [reply apps/src/code-studio/hiddenLessonRedux.js:198]: This is what I get for reviewing PRs early in the morning :)

File: dashboard/config/routes.rb [hunk 2]
```diff
@@ -339,7 +339,7 @@
     # /s/xxx/reset
     get 'reset', to: 'script_levels#reset'
     get 'next', to: 'script_levels#next'
-    get 'hidden_stages', to: 'script_levels#hidden_lesson_ids'
+    get 'hidden_lessons', to: 'script_levels#hidden_lesson_ids'
```
Comment (davidsbailey) [inline dashboard/config/routes.rb:342]: this isn't totally safe -- people who have the page open will get 404s when the hidden_stages route goes away. the safe way would be to keep the old route and add the new route. then take the old route away after 1-2 deploys.

**Review by davidsbailey — APPROVED** [review]
sorry to rain on the parade... lgtm after addressing 1 comment.

**Comment by davidsbailey** [comment]
Woohoo! This is huge!

**Pull request merged**